controller

```diff
@@ -0,0 +1,40 @@
+
+encryption_tutorial.php
```

Comment:
Is this a downloaded library? if so where is it in the repo?

Reply:
No ..Actually .i got code from google  ....and i tried it  in the  codeigniter